### PR TITLE
BatchAI. Fix for VM priority configured via config file.

### PR DESCRIPTION
**Pull request by AlexanderYukhanov**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/6098
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def load_arguments(self, _):
         c.argument('target', options_list=['--target', '-t'], help='Number of nodes which should be allocated immediately after cluster creation. If the cluster is in auto-scale mode, BatchAI can change the number of nodes later based on number of running and queued jobs.')
         c.argument('min_nodes', options_list=['--min'], help='Min nodes count for the auto-scale cluster.', type=int)
         c.argument('max_nodes', options_list=['--max'], help='Max nodes count for the auto-scale cluster.', type=int)
-        c.argument('vm_priority', arg_type=get_enum_type(['dedicated', 'lowpriority']), options_list=['--vm-priority, -p'], help="VM priority. If lowpriority selected the node can get preempted while the job is running.")
+        c.argument('vm_priority', arg_type=get_enum_type(['dedicated', 'lowpriority']), default=None, options_list=['--vm-priority'], help="VM priority. If lowpriority selected the node can get preempted while the job is running.")
```
Comment (tjprescott) [inline src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py]: You shouldn't need to explicitly specify "default=None" unless the command source declares a default.  If the source is a custom method, you should change the default there.

File: src/command_modules/azure-cli-batchai/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.2.2
++++++
+* Now 'az batchai create cluster' respects vm priority configured in the cluster's configuration file.
+* Minor error diagnostics improvement.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-batchai/HISTORY.rst]: Not sure what this means (it sounds like telemetry).  You can probably just remove it.

File: src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py (hunk 1 above)
Comment (AlexanderYukhanov) [reply src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py]: Thanks, fixed

File: src/command_modules/azure-cli-batchai/HISTORY.rst (hunk 2 above)
Comment (AlexanderYukhanov) [reply src/command_modules/azure-cli-batchai/HISTORY.rst]: Agree, removed

**Comment by troydai** [comment]
@AlexanderYukhanov Please fix the style check. The PR is ready to go.

**Pull request merged**